lib/api/post-meta/functions-admin.php

```diff
@@ -104,6 +111,11 @@ function beans_register_post_meta( array $fields, $conditions, $section, $args =
  */
 function _beans_is_post_meta_conditions( $conditions ) {
 
+	// If user has designated boolean true, it's always true. Nothing more to do here.
+	if ( true === $conditions ) {
```

Comment:
Since `$conditions` can be an array or boolean, please note that in the `@param array|bool`